kong/conf_loader.lua

```diff
@@ -1041,7 +1042,7 @@ local function load(path, custom_conf, opts)
   do
     local http_flags = { "ssl", "http2", "proxy_protocol", "transparent",
                          "deferred", "bind", "reuseport", "backlog=%d+" }
-    local stream_flags = { "proxy_protocol", "transparent", "bind",
+    local stream_flags = { "ssl", "proxy_protocol", "transparent", "bind",
```

Comment:
Do we still ship `trasnparent` patch? I though it was gone, and thus needs to be removed everywhere?

Reply:
it is not here anymore: https://github.com/Kong/openresty-patches/tree/fix/remove_mesh/patches/1.15.8.2

Reply:
I created PR to that:
https://github.com/Kong/kong/pull/5350

Reply:
I wasn't sure about the use case of transparent proxy first, but if you said so I'm happy to see it got removed :)